src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,14 @@
 package core.basesyntax;
 
-public class UserService {
+import core.basesyntax.exception.UserNotFoundException;
 
+public class UserService {
     public int getUserScore(String[] records, String email) {
-        return 0;
+        for (String user : records) {
+            if (user.length() != email.length() && user.startsWith(email)) {
```

Comment:
split string with user.split() and save result to String[] splitedUser. After that compare user with equals it's more relieable

Reply:
done